corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -232,6 +222,17 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
         },
     });
 
+    Views.CaseView = Views.CaseViewUnclickable.extend({
```

Comment:
feels weird that CaseView inherits from CaseViewUnclickable, can it be the other way around?